python/ml4ir/applications/classification/model/losses/categorical_cross_entropy.py

```diff
@@ -0,0 +1,35 @@
+# Stating loss functions related to classification use-case.
+from tensorflow.keras import layers
+from tensorflow.keras import losses
+
+from ml4ir.base.model.losses.loss_base import RelevanceLossBase
+from ml4ir.applications.ranking.config.keys import LossKey
+
+
+def get_loss(loss_key) -> RelevanceLossBase:
+    """
+    Factory to get relevance loss related to classification use-case.
+    :param loss_key: LossKey.
+    :return: RelevanceLossBase: corresponding loss.
+    """
+    if loss_key == LossKey.CATEGORICAL_CROSS_ENTROPY:
+        return CategoricalCrossEntropy()
+    else:
+        raise NotImplementedError
+
+
+class CategoricalCrossEntropy(RelevanceLossBase):
+
+    def get_loss_fn(self, **kwargs):
+        """
+        Define a softmax cross entropy loss
+        """
+        cce = losses.CategoricalCrossentropy(reduction=losses.Reduction.SUM_OVER_BATCH_SIZE)
```

Comment:
@Ullimague Would be good to check if you want to set this particular reduction. Usually, if your loss value is very high, you can change it to mean.

Reply:
Good point, at some point we will want to make it parametrizable, not this highest priority for this pr.